From 856f9e20c456664600e112e5735d22813f8c9e57 Mon Sep 17 00:00:00 2001
From: Richard Gong <richard.gong@intel.com>
Date: Wed, 12 Feb 2020 11:57:56 -0600
Subject: [PATCH 035/120] fpga: stratix10-soc: add compatible property value
 for intel agilex

commit d843b93a842c8e32b78ba1e5f0cdfd3f001c38e2 from
https://github.com/altera-opensource/linux-socfpga.git

[upstream commit 401e62675144494fda178b1985a530cfe3f4b78f]

Add compatible property value so we can reuse FPGA manager driver on
Intel Agilex SoC platform.

Signed-off-by: Richard Gong <richard.gong@intel.com>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 drivers/fpga/stratix10-soc.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/fpga/stratix10-soc.c b/drivers/fpga/stratix10-soc.c
index bae98c8ce362..44b7c569d4dc 100644
--- a/drivers/fpga/stratix10-soc.c
+++ b/drivers/fpga/stratix10-soc.c
@@ -477,7 +477,8 @@ static int s10_remove(struct platform_device *pdev)
 }
 
 static const struct of_device_id s10_of_match[] = {
-	{ .compatible = "intel,stratix10-soc-fpga-mgr", },
+	{.compatible = "intel,stratix10-soc-fpga-mgr"},
+	{.compatible = "intel,agilex-soc-fpga-mgr"},
 	{},
 };
 
-- 
2.31.1

